campaignion_email_to_target/campaignion_email_to_target.module

```diff
@@ -134,16 +134,18 @@ function campaignion_email_to_target_token_info() {
     try {
       $api = Container::get()->loadService('campaignion_email_to_target.api.Client');
       $dataset = $api->getDataset($ds_name);
-      $type = 'email-to-target';
-      $info['types'][$type] = [
-        'name' => t('Email-to-Target dataset: %name', ['%name' => $dataset->title]),
-        'description' => t('Replace values based on the selected target(s).'),
-      ];
-      foreach ($dataset->attributes as $attribute) {
-        $info['tokens'][$type][$attribute->key] = [
-          'name' => $attribute->title,
-          'description' => $attribute->description,
+      $types = ['email-to-target', 'target'];
```

Comment:
Should we keep the documetation of the `email-to-target` tokens? If so, should we add an info that it’s deprecated?